SPECS/fsverity-utils/fsverity-utils.spec

```
@@ -0,0 +1,118 @@
+Summary:        fsverity utilities
+Name:           fsverity-utils
+Version:        1.5
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/
+Source0:        https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git/snapshot/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  glibc-headers
+BuildRequires:  kernel-headers
+BuildRequires:  make
+BuildRequires:  openssl-devel
+Requires:       libfsverity = %{version}-%{release}
+
+%description
+This is fsverity, a userspace utility for fs-verity.
+fs-verity is a Linux kernel feature that does transparent on-demand
+integrity/authenticity verification of the contents of read-only files,
+using a hidden Merkle tree (hash tree) associated with the file.
+The mechanism is similar to dm-verity, but implemented at the file level
+rather than at the block device level. The fsverity utility allows you
+to set up fs-verity protected files.
+
+%package -n libfsverity
+Summary:        Library files for fsverity-utils
+
+%description -n libfsverity
+Library for fsverity-utils.
+
+%package devel
+Summary:        Development package for fsverity-utils
+Requires:       %{name} = %{version}-%{release}
+Requires:       libfsverity = %{version}-%{release}
+
+%description devel
+Development package for fsverity-utils. This package includes the
+libfsverity header and library files.
+
+%prep
+%autosetup -p1
+
+%build
+%make_build CFLAGS="$CFLAGS -g" USE_SHARED_LIB=1
+
+%install
+%make_install PREFIX=%{_prefix} LIBDIR=%{_libdir}  CFLAGS="$CFLAGS -g" USE_SHARED_LIB=1
```

Nit: `-g` is already part of our default set of compiler flags.

```suggestion
%make_install PREFIX=%{_prefix} LIBDIR=%{_libdir} USE_SHARED_LIB=1
```

---

Done